tests/testthat/test-default-scoring-rules.R

```diff
@@ -0,0 +1,71 @@
+test_that("`select_rules` works as expected", {
+
+  expect_equal(
+    scoringutils:::select_rules(rules_point(), select = "all"),
+    rules_point()
+  )
+
+  expect_equal(
+    names(scoringutils:::select_rules(rules_point(), select = "ape")),
```

Comment:
this seems like quite a nice workflow for users?

Reply:
but they can just do `rules_point(select = "ape")` instead, which is more concise. 

I can see a case once you start combining several lists, e.g. `select_rules(a(), b(), select = c("c", "d").

Reply:
> I can see a use case once you start combining several lists,

This would be the argument and the push back on it being more concise is that yes that is true but then each function is doing multiple things which can confuse readers. I don't have an extremely strong opinion but think if should be exported and if keeping as is then should at least make use of match.arg or the rlang version) to get some functionality.